libpod/version.go

```diff
@@ -38,11 +42,31 @@ func GetVersion() (Version, error) {
 			return Version{}, err
 		}
 	}
+
+	conmonOutput, err := rt.GetConmonVersion()
+	if err != nil {
+		return Version{}, err
+	}
+	runcOutput, err := rt.GetRuncVersion()
+	if err != nil {
+		return Version{}, err
+	}
+
 	return Version{
 		Version:   podmanVersion.Version,
 		GoVersion: runtime.Version(),
 		GitCommit: GitCommit,
 		Built:     buildTime,
 		OsArch:    runtime.GOOS + "/" + runtime.GOARCH,
+		Conmon:    strings.TrimSuffix(strings.Replace(conmonOutput, "\n", ", ", 1), "\n"),
+		RunC:      strings.TrimSuffix(runcOutput, "\n"),
 	}, nil
 }
+
+func (r *Runtime) GetConmonVersion() (string, error) {
```

Comment:
Should this be private?

Also, should we push the string processing happening above in the `return Version` into here?

Reply:
done